lib/install.go

```diff
@@ -268,6 +269,7 @@ func InstallProductVersion(product Product, dryRun bool, version, customBinaryPa
 			os.Exit(1)
 		}
 	}
+	return nil
```

Comment:
I guess it's best to leave a comment that `os.Exit()` is to be changed to `return` probably.

Reply:
The reason we are doing this is because we want our libraries to be accessible by other users. Exit probably does not provide enough info to users using our library. 

However, we may need to Exit(1), so bots that are using our tool knows that something went wrong. I suggest we Exit(1) at the parent function that is calling these functions

Reply:
This should be done :)